src/components/PurchaseOrder/getPOActionMenu.js

```diff
@@ -104,17 +108,27 @@ export function getPOActionMenu({
           </Button>
         )}
       </IfPermission>
-      <IfPermission perm="ui-orders.order.reopen">
-        {isReopenButtonVisible && (
-          <Button
-            buttonStyle="dropdownItem"
-            data-test-reopen-order-button
-            onClick={clickReopen}
-          >
-            <FormattedMessage id="ui-orders.paneBlock.reopenBtn" />
-          </Button>
-        )}
+      <IfPermission perm="orders.item.post">
+        <Button
+          buttonStyle="dropdownItem"
+          data-test-clone-order-button
+          onClick={clickClone}
+        >
+          <FormattedMessage id="ui-orders.paneBlock.cloneBtn" />
```

Comment:
Looks like `<Icon size="small" icon="duplicate">` could be used here

Reply:
I wouldn't rush with using icons, we have a lot of actions without them.